src/Storages/MergeTree/MergeTreeWriteAheadLog.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <DataStreams/NativeBlockInputStream.h>
+#include <DataStreams/NativeBlockOutputStream.h>
+#include <Storages/MergeTree/IMergeTreeDataPart.h>
+#include <Disks/IDisk.h>
+
+namespace DB
+{
+
+class MergeTreeData;
+
+class MergeTreeWriteAheadLog
```

Comment:
Comment, what we store in this log? In which format (how record for a single part looks like)? How it can be restored from disk?